### PR TITLE
Quick fix for seeds results

### PR DESCRIPTION
**Pull request by mphprogrammer**
InstitutionBuilder.run returns a results hash, with the version record as a member. Adjusted seeds to compensate.

Added HOST to application.yml.example

### CODE REVIEW

File: config/application.yml.example [hunk 1]
```diff
@@ -2,3 +2,4 @@ ADMIN_EMAIL: 'something...'
 ADMIN_PW: 'something...'
 SECRET_KEY_BASE: 'something ...'
 LINK_HOST: 'http://localhost:3000' # https://api.vets.gov
+HOST 'localhost'
```
Comment (ben-USDS-damman) [inline config/application.yml.example]: What's this HOST var?

**Review by ben-USDS-damman — APPROVED** [review]
LGTM but what is the HOST ENV var for?

File: config/application.yml.example (hunk 1 above)
Comment (mphprogrammer) [inline config/application.yml.example]: The serializer specs were braking without them.

Comment (mphprogrammer) [inline config/application.yml.example]: Wait and now they are not. Strange.

Comment (mphprogrammer) [inline config/application.yml.example]: @ben-USDS-damman Removed. Its strange.

**Comment by mphprogrammer** [comment]
Thanks...

**Pull request merged**